oci/oci.go

```diff
@@ -287,7 +287,15 @@ func (o *Oci) Tag(existingRef, newRef string) error {
 
 	descriptor, ok := refs[existingRef]
 	if !ok {
-		return errors.Errorf("policy [%s] not found in the local store", existingRef)
+		for _, v := range refs {
+			if strings.Contains(v.Digest.String(), existingRef) {
```

Comment:
Is `strings.Contain` sufficient or do we need a stronger regex match?
What's the format of `v.Digest.String()`?

Reply:
`v.Digest.String()` is the manifest SHA256 string formatted as `sha256:<digest_string>` and the image reference id is formed from the first 11 chars of the digest string, so we could do a direct comparison with the first chars instead of a strings.Contains or use strings.HasPrefix. 

Maybe using strings.HasPrefix makes more sense in this scenario, would you agree ?